target/linux/ath79/dts/ar7242_ubnt_edgeswitch-8xp.dts

```diff
@@ -0,0 +1,306 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7242_ubnt_sw.dtsi"
+
+/ {
+	compatible = "ubnt,edgeswitch-8xp", "qca,ar7242";
+	model = "Ubiquiti EdgeSwitch 8XP";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_usr;
+		led-failsafe = &led_usr;
+		led-running = &led_usr;
+		led-upgrade = &led_usr;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_usr: usr {
+			label = "ubnt:yellow:usr";
+			gpios = <&gpio 13 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	spi {
```

Comment:
Please choose a different name to distinguish from &spi.
E.g. https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9561_tplink_archer-c25-v1.dts#L25